chamber/models/patch.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import unicode_literals
+
+from django.db.models.fields import Field
+
+
+def field_init(self, *args, **kwargs):
+    """
+    Patches a Django Field's `__init__` method for easier usage of optional `kwargs`. It defines a `humanized` attribute
+    on a field for better display of its value.
+    """
+    humanize_func = kwargs.pop('humanized', None)
+    if humanize_func:
+        def humanize(val, inst, *args, **kwargs):
+            return humanize_func(val, inst, field=self, *args, **kwargs)
+        self.humanized = humanize
+    else:
+        self.humanized = self.default_humanized
+    getattr(self, '_init_chamber_patch_')(*args, **kwargs)
+
+
+Field.default_humanized = None
+setattr(Field, '_init_chamber_patch_', Field.__init__)
```

Comment:
Problém ale je, že ty stejně měníš protected outside of instance, jen to děláš přes setattr, to že ti neřve PEP je sice fajn, ale děláš stejně špatnou věc, jen ti to nepodchytil linter. Tak proč nemít hezčí syntaxi a vypnout warning pro ten řádek?